### PR TITLE
Added "-part1" in partitionPath for multipath

### PR DESCRIPTION
**Pull request by mattcui**
This PR is for addressing [issue 92](https://github.com/cloudfoundry/bosh-agent/issues/92). I added the unit test and verified in my testing CF deployment. @cppforlife, please help view. Thanks.

/cc @maximilien @dpb587-pivotal

### CODE REVIEW

**Comment by mingxiao** [comment]
This is a PR against master, which we do not allow. We will verify and merge onto develop.

**Comment by mingxiao** [comment]
Merged onto develop, waiting on integration tests.

File: platform/disk/sfdisk_partitioner_test.go [hunk 1]
```diff
@@ -136,6 +145,20 @@ var _ = Describe("sfdiskPartitioner", func() {
 		Expect(len(runner.RunCommandsWithInput)).To(Equal(0))
 	})
 
+	It("sfdisk partition when partitions already match for mutlitpath", func() {
```
Comment (zaksoup) [inline platform/disk/sfdisk_partitioner_test.go]: typo: `"mutlitpath"`
/cc @maximilien

Comment (mattcui) [inline platform/disk/sfdisk_partitioner_test.go]: Sorry, I submitted another [PR 94](https://github.com/cloudfoundry/bosh-agent/pull/94) to correct it.

**Pull request closed without merging**